### PR TITLE
Single operand image. Fixes #1396

### LINKED ISSUE #1396: Single operand image
Since an openjdk/ubi8 image is available for ppc64le and s390x there's no more need to refer different operand images in: https://github.com/infinispan/infinispan-operator/blob/dbaf3a6fc98eea26c8c3a604f055dc73f17697ab/controllers/constants/image.go#L18

see also https://issues.redhat.com/browse/JDG-5110

### PR DESCRIPTION
**Pull request by rigazilla**
Fixes #1396

### CODE REVIEW

File: controllers/constants/image.go [hunk 1]
```diff
@@ -1,33 +1,18 @@
...
-		return GetEnvWithDefault(OperandImageOpenJ9, DefaultOperandImageOpenJ9)
-	default:
-		// Try new name RELATED_IMAGE_OPENJDK first
-		retVal := GetEnvWithDefault(OperandImageOpenJDK, "")
-		if retVal != "" {
-			return retVal
-		}
-		// If RELATED_IMAGE_OPENJDK is not defined fallback on old name for backward comp
-		return GetEnvWithDefault("DEFAULT_IMAGE", DefaultOperandImageOpenJDK)
+	if retVal := GetEnvWithDefault(OperandImageOpenJDK, ""); retVal != "" {
```
Comment (ryanemerson) [inline controllers/constants/image.go:13]: As we're not using a default value here, we can just do:

```go
	if retVal, exists := os.LookupEnv(OperandImageOpenJ9); exists {
		return retVal
	}
	return GetEnvWithDefault("DEFAULT_IMAGE", DefaultOperandImageOpenJDK)
```

Comment (ryanemerson) [reply controllers/constants/image.go:13]: Ignore that comment :slightly_smiling_face:. `os.LookupEnv` will return true if the env is set but empty, which is different to `GetEnvWithDefault`.

Comment (rigazilla) [reply controllers/constants/image.go]: ok, done

**Comment by ryanemerson** [comment]
Thanks @rigazilla

**Pull request merged**